src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -3929,7 +3929,7 @@ private void LE_Handler(Instruction instruction)
                 {
                     double lhs = opdata2.IsDouble ? opdata2.DoubleValue: opdata2.IntegerValue;
                     double rhs = opdata1.IsDouble ? opdata1.DoubleValue: opdata1.IntegerValue;
-                    opdata2 = StackValue.BuildBoolean(MathUtils.IsLessThanOrEquals(lhs, rhs));
+                    opdata2 = StackValue.BuildBoolean(lhs <= rhs);
```

Comment:
as above

Reply:
Okay, just got to read the comments on the PR itself, and see why this is being done.